docs/Using-Pantheon/Debugging.md

```diff
@@ -52,26 +55,72 @@ block of the `prometheus.yml` file:
     --rpc-http-cors-origins="all" --rpc-http-enabled --metrics-enabled
     ```
 
-4. In another terminal, run Prometheus specifying the `prometheus.yml` file: 
+1. In another terminal, run Prometheus specifying the `prometheus.yml` file: 
 
     ```bash tab="Example"
     prometheus --config.file=config.yml 
     ```
 
-5. Open a web browser to `http://localhost:9090` to view the Prometheus graphical interface.
+1. Open a web browser to `http://localhost:9090` to view the Prometheus graphical interface.
 
-6. Choose **Graph** from the menu bar and click the **Console** tab below.
+1. Choose **Graph** from the menu bar and click the **Console** tab below.
 
-7. From the **Insert metric at cursor** drop-down, select a metric such as `pantheon_blockchain_difficulty_total` or
+1. From the **Insert metric at cursor** drop-down, select a metric such as `pantheon_blockchain_difficulty_total` or
 `pantheon_blockchain_height` and click **Execute**. The values are displayed below.
 
     Click the **Graph** tab to view the data as a time-based graph. The query string is displayed below the graph. 
     For example: `{pantheon_blockchain_height{instance="localhost:9545",job="prometheus"}`
+
+### Running Prometheus with Pantheon in Push Mode 
+
+By default, Prometheus polls Pantheon for metrics but sometimes metrics are hard to poll (for example, 
+when running inside Docker containers with varying IP addresses). The `push` mode supports Pantheon 
+pushing metrics to a [Prometheus Pushgateway](https://github.com/prometheus/pushgateway).   
+
+To configure Prometheus and run with Pantheon pushing to a push gateway: 
+
+1. Configure Prometheus for Pantheon to push to the push gateway. For example, add the following yaml fragment to the `scrape_configs`
+   block of the `prometheus.yml` file:
+    
+       ```yml tab="Example"
+        job_name: pantheon-dev
+            metrics_path: /metrics
+            scheme: http
+            static_configs:
+            - targets:
+              - localhost:9091
+       ```
+   
+    !!! note
+        The [`--host-whitelist` option](../Reference/Pantheon-CLI-Syntax.md#host-whitelist) defaults to `localhost`.
+        If `127.0.0.1` is specified instead of `localhost` in the `prometheus.yml` file, add `127.0.0.1` to the host whitelist
+        using [`--host-whitelist`](../Reference/Pantheon-CLI-Syntax.md#host-whitelist) when starting Pantheon. 
+
+1. Start the push gateway. The push gateway can be deployed using the Docker image: 
+
+    ```bash tab="Example"
+    docker pull prom/pushgateway
+    docker run -d -p 9091:9091 prom/pushgateway
+    ```
+
+1. Start Pantheon specifying the metrics mode is `push` and port of the push gateway: 
+
+    ```bash tab="Example"
+    pantheon --network=dev --miner-enabled --miner-coinbase fe3b557e8fb62b89f4916b721be55ceb828dbd73 --rpc-http-cors-origins="all" --rpc-http-enabled --metrics-enabled --metrics-port=9091 --metrics-mode=push
```

Comment:
CLI changed in #678 , `--metrics-port=9091` changes to `--metrics-push-port=9091`

Also, we should add `--metrics-push-host=localhost`.  Even though it's bascaily the default when they cut and paste they will have the correct CLI to change the push host name.

Reply:
Done

Reply:
I've changed the port but Pantheon doesn't accept `--metrics-push-host=localhost` on the command line. I tried `--metrics-push-host="localhost"` as well. Should it?